### ISSUE TITLE
Support for project manager?

### ISSUE THREAD

**Issue by chmac** [zokugun/vscode-sync-settings#11]
Are you open to supporting syncing of the `projects.json` file for the project manager extension? It has ~2.2m installs and it was explicitly supported by the shanshan sync.

https://github.com/alefragnani/vscode-project-manager

It stores it's projects list in a `projects.json` file which is at `~/Library/Application Support/Code/User/globalStrorage/alefragnani/vscode-project-manager/projects.json`.

What about adding the option to specify arbitrary paths relevant to the VSCode user directory? That would mean I could add a line which says I want to sync `globalStrorage/alefragnani/vscode-project-manager/projects.json` and then it gets updated.

**Comment by chmac** [comment]
For anyone who finds this and wants to make it work today, I'm using this workaround:

- Move the `projects.json` file into `~/Library/ApplicationSupport/Code/User/globalStorage/zokugun.sync-settings/repository`
- Symlink it back to its original locaion
- `git add`, `git commit`, `git push` (you might need to set an upstream branch)

Then on every other machine you need to repeat the symlink piece.

**Comment by daiyam** [comment]
I could add a list of files in VSCode' settings that need to be synced, like:
```
"syncSettings.additionalFiles": [
    "~globalStrorage/alefragnani/vscode-project-manager/projects.json",
    "~/vscode_projects.json",
]
```

**Comment by chmac** [comment]
@daiyam Yeah, that would be an awesome option. Especially supporting `~/` so that the paths would be portable across many machines. That would be great.

**Comment by daiyam** [comment]
@chmac Can you try: [sync-settings-0.8.1.vsix](https://github.com/zokugun/vscode-sync-settings/files/8170559/sync-settings-0.8.1.vsix.zip)?

The settings should be:
```
"syncSettings.additionalFiles": [
    "~globalStorage/alefragnani.project-manager/projects.json",
],
```

**Comment by chmac** [comment]
@daiyam I'm away from one of my computers so I can't test it out until early next week.

**Comment by chmac** [comment]
@daiyam I wasn't able to install the 0.8.1 file, but it looks like this feature shipped in 0.9.0. I've tested it on my setup and it works great. Thanks a lot for adding this.

🧡 🧡 🧡
